src/dune/super_context.ml

```diff
@@ -1,8 +1,8 @@
 open! Stdune
 open Import
 
-let default_context_flags (ctx : Context.t) =
-  let c = Ocaml_config.ocamlc_cflags ctx.ocaml_config in
+let default_context_flags (_ctx : Context.t) =
```

Comment:
We should simplify (or even delete) this function. We'll restore it if we need to. It's better to keep the code lean, otherwise if we don't end up pushing further we end up with historical code.

That being said, it seems that the `-std` filtering should go somewhere else. Otherwise we might end up passing to the C++ compiler again.